docs/metadata.md

```diff
@@ -392,22 +400,41 @@ schema = metadata.MetadataSchema(
                 "binaryFormat": "10p",
                 "pattern": "^([1-9][0-9]{3})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])?$",
             },
+            "phenotype": {
+                "description": "Phenotypic measurements on this individual",
+                "type": "object",
+                "properties": {
+                    "height": {
+                        "description": "Height in metres, or NaN if unknown",
+                        "type": "number",
+                        "binaryFormat": "f",
+                    },
+                    "age": {
+                        "description": "Age in years at time of sampling, or -1 if unknown",
+                        "type": "number",
+                        "binaryFormat": "h",
+                    },
+                },
+                "default": {"height": float("NaN"), "age": -1},
```

Comment:
I think these should be done as separate defaults for age and height, then with an empty object default for phenotype. This allows either both, or one to be missing.

Reply:
Yes, I can see that, but you said previously that it wasn't obvious that you could set a top-level dict of defaults like this, so I thought it was worth showing an example.